backend/services/agency_store.py

```
@@ -0,0 +1,151 @@
+                if cand.strip().lower() == k.lower():
+                    return (row[cand] or "").strip()
+        return default
+
+    due = g(["Due Date","DueDate"])
+    try:
+        # Normalize to YYYY-MM-DD
+        if due:
+            dt = datetime.datetime.fromisoformat(due.replace("/", "-"))
+            due = dt.date().isoformat()
+    except Exception:
+        pass
```

Ruff (0.13.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)